### DISCUSSION TITLE
Pimox Scripts

### DISCUSSION THREAD

**Discussion by asylumexp** [BramSuurdje/proxmox-helper-scripts#35] [General]
Hey,

just wanted to see if you were fine with me using this for [pimox-scripts](https://github.com/AsylumEXP/Proxmox), since this was such a nice website which isn’t in use anymore.

Thanks!

**Comment by bramsuurdje** [comment]
absolutely brother, even though it is still in use.

**Reply by asylumexp** [reply]
ah yeah i meant that the main domain was being directed to the old github pages site.

thank you though!
